apax/md/schedules.py

```diff
@@ -17,13 +17,16 @@ def __call__(self, step) -> float:
 
 
 class PieceWiseLinearTSchedule(TSchedule):
-    def __init__(self, T0: int, values: list[float], steps: list[int]):
+    def __init__(self, T0: int, temperatures: list[float], durations: list[int]):
         self.T0 = T0
-        self.values = jnp.array(values)
+        self.temperatures = jnp.array(temperatures)
+        steps = np.cumsum(durations)
         self.steps = jnp.array(steps)
 
     def __call__(self, step) -> float:
-        T = jnp.interp(step, self.steps, self.values, left=self.T0, right=self.values[-1])
+        T = jnp.interp(
+            step, self.steps, self.temperatures, left=self.T0, right=self.temperatures[-1]
+        )
         return T * units.kB
```

Comment:
You made the changes to MD config also in https://github.com/apax-hub/apax/pull/352 but the changes in this file are missing. Can this cause issues if https://github.com/apax-hub/apax/pull/352 is merged first? Tests should cover this?

Reply:
Yes the test does actually fail rn but should be fixed with the other PR

Reply:
Not sure what I was thinking here